lib/utils.js

```diff
@@ -110,27 +108,6 @@ const processOption = (options, config, hasWhitespace) => {
   }
 }
 
-// Restart LogDNA Agent
```

Comment:
@darinspivey, can you remove the references to these `restartSelf` and `updateSelf` methods from `README.md` and some other places as well?

Reply:
yes, good catch

Reply:
Removed one line about auto-upgrading.